frontend/src/components/workspaces/Templates/TemplatesTableRow/TemplatesTableRow.tsx

```diff
@@ -89,13 +90,13 @@ const TemplatesTableRow: FC<ITemplatesTableRowProps> = ({ ...props }) => {
 
   const createInstanceHandler = () => {
     setCreateDisabled(true);
-    createInstance(id)
+    createInstance(template.id)
       .then(() => {
         refreshClock();
         setTimeout(() => {
           setCreateDisabled(false);
         }, 400);
```

Comment:
```suggestion
        setTimeout(setCreateDisabled, 400, false);
```
Shorten this as we're around here 😁